### ISSUE TITLE
Taskbar is still there in full screen mode entered from maximized window

### ISSUE THREAD

**Issue by kingcean** [microsoft/WindowsAppSDK#2577]
### Describe the bug

The taskbar is not covered if the window was maximized before entering full screen.

### Steps to reproduce the bug

1. Maximize the window.
2. Enter full screen.

### Expected behavior

The taskbar should disappear until the window closes or exits from full screen.

### Screenshots

![Maximize window](https://user-images.githubusercontent.com/1724940/172054901-6a4696f1-f7c5-4a24-b014-9f2568f55d2d.jpg)
↑ Maximize window

![Full screen](https://user-images.githubusercontent.com/1724940/172054931-b969acea-55ab-4510-b1aa-56c06b2d06e0.jpg)
↑ Then enter full screen (but you can still see the taskbar)

### NuGet package version

1.1.0

### Packaging type

Packaged (MSIX)

### Windows version

Windows 11 version 21H2 (22000)

### IDE

Visual Studio 2022

**Comment by marb2000** [comment]
Hey @kingcean do you have a repro so we can take a look at the APIs that you are using?

**Comment by kingcean** [comment]
Yes. You can maximize the window first (by programming or by clicking/tapping the caption button), and then enter full screen to have a look.

```csharp
// var appWindow = ... The AppWindow instance of MainWindow.
var presenter = appWindow.Presenter as Microsoft.UI.Windowing.OverlappedPresenter;
presenter.Maximize();
appWindow.SetPresenter(Microsoft.UI.Windowing.AppWindowPresenterKind.FullScreen);
```

**Comment by riverar** [comment]
This repros 100% if window is in maximized state before fullscreen presenter is set.

Repro project:
[was_issue_2577.zip](https://github.com/microsoft/WindowsAppSDK/files/8855057/was_issue_2577.zip)

**Comment by dotMorten** [comment]
I can repro this with the WinUIEx Sample app. Just run the app, maximize window, then select the "Fullscreen" presenter in the presenter dropdown. My app restores to the non-maximized state when I do this, yet claims to be fullscreen.

**Comment by rkarman** [comment]
Yeah, this one should not have been able to slip through the automated tests. 😞  I've logged a task for the dev team and we'll be adding it to a servicing release as soon as we have a fix.

MSFT dev team tracking: http://task.ms/39908320

**Comment by kingcean** [comment]
I can still repro this on Windows App SDK 1.1.1.

**Comment by koenvd** [comment]
Still reproducible with Windows App SDK 1.1.4.

**Comment by stevewri** [comment]
Thanks! 😊

**Comment by kingcean** [comment]
Fixed - v1.2.
